app/src/main/java/com/practicum/playlistmaker/data/App.kt

```diff
@@ -0,0 +1,37 @@
+package com.practicum.playlistmaker.data
+
+import android.app.Application
+import android.content.res.Configuration
+import com.practicum.playlistmaker.Creator.Creator
+import com.practicum.playlistmaker.domain.api.SettingsThemeInteractor
+
+class App : Application() {
+
+    companion object {
+        const val THEME_SWITCHER_PREFERENCES = "theme_switcher_preferences"
+        const val APP_THEME_KEY = "theme_key"
+    }
+
+    var darkTheme = false
```

Comment:
Данную переменную лучше сделать приватной, либо сделать ей приватный сеттер:
```suggestion
    var darkTheme = false
       private set
```
Т.к. предполагается, что значение этой переменной менять должен только этот класс при смене темы